### PR TITLE
Use submit button label of form for captcha submit button [MAILPOET-4496]

### PR DESCRIPTION
**Pull request by websupporter**
## Description

N/A

## QA notes
Simple steps to test:
1. Make sure you have some form already created (MailPoet > Forms)
2. In a private browsing window, go to the front-end of your test site and sign up to a form
3. After subscribing and seeing "Confirm you’re not a robot" page, make sure the Subscribe button is the same as in the form you were subscribing to
4. Feel free to change the text of the Subscribe button and test again
5. Check in different site language

## Linked tickets

Fixes [MAILPOET-4496]

[MAILPOET-4496]: https://mailpoet.atlassian.net/browse/MAILPOET-4496?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/tests/integration/Subscription/CaptchaRendererTest.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    $captchaSession->setFormData(['form_id' => $form->getId()]);
+
+    $testee = $this->diContainer->get(CaptchaRenderer::class);
+    $result = $testee->getCaptchaPageContent($captchaSession->getId());
+    $this->assertStringContainsString('value="Subscribe"', $result);
+  }
+
+  public function _before() {
+    $this->diContainer->get(FormsRepository::class)->truncate();
+    parent::_before(); // TODO: Change the autogenerated stub
```
Comment (costasovo) [inline mailpoet/tests/integration/Subscription/CaptchaRendererTest.php]: This to-do comment was probably generated by IDE. Could you please remove it as it seems to be outdated and could be confusing?

**Review by costasovo — CHANGES_REQUESTED** [review]
Looks good! Thank you for adding the tests! I'm requesting changes just because a forgotten Todo comment.

File: mailpoet/tests/integration/Subscription/CaptchaRendererTest.php (hunk 1 above)
Comment (websupporter) [reply mailpoet/tests/integration/Subscription/CaptchaRendererTest.php]: Fixed

**Comment by websupporter** [comment]
Thank you for your review. I fixed the outstanding issue @costasovo

**Review by costasovo — APPROVED** [review]

**Review by Aschepikov — APPROVED** [review]
This is verified, good job!

**Pull request merged**